app/api/bedrock.ts

```
@@ -0,0 +1,259 @@
+      return NextResponse.json(parsed);
+    }
+
+    // Handle streaming response
+    const transformedStream = transformBedrockStream(res.body, modelId);
+    const encoder = new TextEncoder();
+    const stream = new ReadableStream({
+      async start(controller) {
+        try {
+          for await (const chunk of transformedStream) {
+            // Ensure we're sending non-empty chunks
+            if (chunk && chunk.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)